tests/FSharp.Compiler.ComponentTests/Miscellaneous/MigratedTypeCheckTests.fs

```diff
@@ -0,0 +1,501 @@
+// Copyright (c) Microsoft Corporation.  All Rights Reserved.  See License.txt in the project root for license information.
+module FSharp.Compiler.ComponentTests.Miscellaneous.FsharpSuiteMigrated.TypeCheckTests
+
+open Xunit
+open FSharp.Test
+open FSharp.Test.ScriptHelpers 
+open TestFrameworkAdapter  
+
+[<Fact>]
+let misc () = singleTestBuildAndRunVersion "typecheck/misc" FSC_OPTIMIZED LangVersion.SupportsMl
+
+[<FactForDESKTOP(Skip = "Failing in new test framework")>]
```

Comment:
Everything which was failing (e.g. diagnostics were slightly different) is skiped here and **continues to exist** in the previous Fsharp Test Suite.